src/ceph-volume/ceph_volume/util/prepare.py

```diff
@@ -221,7 +226,7 @@ def osd_mkfs_bluestore(osd_id, fsid, keyring=None, wal=False, db=False):
 
     command = base_command + supplementary_command
 
-    process.run(command, obfuscate='--key')
+    process.run(command, obfuscate='--keyfile')
```

Comment:
if the `temp` file is never closed it will not be removed. Are we OK with the file being around? If we want to remove it we would need to check if `temp` was created, and if so, then close it after `process.run` here